code_schemes/kakuma_s01e01_reasons.json

```diff
@@ -0,0 +1,124 @@
+{
+  "SchemeID": "Scheme-369bb070",
+  "Name": "KAKUMA S01E01 Reasons",
```

Comment:
KAKUMA is in caps but Dadaab was not. Please pick one and standardise

Reply:
going with CAPs for all since other pipelines use caps too. Thanks